lib/presentation/util/toast_message_handler.dart

```diff
@@ -266,6 +269,20 @@ class ToastMessageHandler {
     });
   }
 
+  void _handleDeleteSharedSpaceMembersToastMessage(BuildContext context, DeleteSharedSpaceMembersState deleteSharedSpaceMembersState) {
+    deleteSharedSpaceMembersState.viewState.fold((failure) {
+      if (failure is DeleteSharedSpaceMemberFailure) {
+        appToast.showErrorToast(AppLocalizations.of(context).user_could_not_be_removed);
+        _cleanDeleteSharedSpaceMembersViewState();
+      }
+    }, (success) {
+      if (success is DeleteSharedSpaceMemberViewState) {
+        appToast.showToast(AppLocalizations.of(context).user_is_successfully_removed);
```

Comment:
why the toast on your GIF have the color is white?

Reply:
Pls refer this: https://developer.android.com/about/versions/11/behavior-changes-11#toasts
I recorded demo video on my pixel phone (Android 11)

Reply:
It's the correct background color on the lower OS (verified on Android 9.0 simulator)